### ISSUE TITLE
train a classifier model using `ConvNext` to classify dog breeds by @pavaris-pm

### ISSUE THREAD

**Issue by pavaris-pm** [pavaris-pm/dog-breed-classification-webapp#6]
For all contributors here @supermind-c  @ToydnyChristini @RyuNumchon  📢 ❗  

Even it seems to be great on 92% validation accuracy with `ConvNext` architecture, @supermind-c  has made a visualization of my torch training loop, and seems like my code for display the training statistics (e.g. training acc, training loss) is wrong in the colab notebook. Since it is my logical error not a syntax error, i will fix it and give you a link in this separated PR so that you won't be confused with #1  which is the main issues🔥

**Comment by pavaris-pm** [comment]
@supermind-c  i think i've got it, can you please run this [colab notebook](https://colab.research.google.com/drive/1jm1e51rN70tJN77wAJ_gsmSo8qaj0Y7g?usp=sharing). i think we can push the model to the huggingface hub at this time. It's time for your A100 👍🏻

**Comment by pavaris-pm** [comment]
> @supermind-c i think i've got it, can you please run this [colab notebook](https://colab.research.google.com/drive/1jm1e51rN70tJN77wAJ_gsmSo8qaj0Y7g?usp=sharing). i think we can push the model to the huggingface hub at this time. It's time for your A100 👍🏻

The model already published in huggingface hub already ! Thanks for all collaborators 🎊
